nautobot/extras/models/relationships.py

```diff
@@ -736,3 +774,65 @@ def _validate_relationship_filter_restriction(self):
                 raise ValidationError(
                     {side_name: (f"{side} violates {self.relationship} {side_name}_filter restriction")}
                 )
+
+
+def get_relationships_errors(request, obj, output_for="ui"):
+    required_relationships = Relationship.objects.get_required_for_model(obj)
+    relationships_errors = []
+    for side, relations in required_relationships.items():
+        for relation in relations:
+
+            if relation.skip_required(obj, side):
+                continue
+
+            required_model_class = getattr(relation, f"{side}_type").model_class()
+
+            if required_model_class.objects.count() == 0:
+                model_meta = required_model_class._meta
+                required = model_meta.verbose_name
+
+                if relation.type in [
+                    RelationshipTypeChoices.TYPE_ONE_TO_MANY,
+                    RelationshipTypeChoices.TYPE_MANY_TO_MANY,
+                ]:
+                    num_required_verbose = "at least one"
+                else:
+                    num_required_verbose = "a"
+
+                if output_for == "ui":
+                    try:
+                        add_url = reverse(get_route_for_model(required_model_class, "add"))
+                        add_message = f"<a href='{add_url}'>Click here</a> to create a {model_meta.verbose_name}."
+                    except NoReverseMatch:
+                        add_message = f"You need to create a {required} first"
+
+                    relationships_errors.append(
+                        f"{obj._meta.verbose_name_plural.capitalize()} require {num_required_verbose} {required}, "
```

Comment:
`capitalize()` will change `VLANs` to `Vlans`, `IP addresses` to `Ip addresses`, etc., which is undesirable. Can we reword this to avoid the need to call `capitalize()`?